File: JAVA/CodeFights/4_adjacentElementsProduct/code.java
int adjacentElementsProduct(int[] inputArray) {
    int result = -1000;
    int result_temp = 0;
    int longitud = inputArray.length;
    for (int i = 0; i <= (longitud-2); i++)
    {
        result_temp = inputArray[i] * inputArray[i+1];
        if (result < result_temp)
        {
            result = result_temp;
        }
    }
    return result;

File: PYTHON3/CodeFights/4_adjacentElementsProduct/code.py
def adjacentElementsProduct(inputArray):
    result = -1000
    result_temp = 0
    longitud = len(inputArray)
    longitud-=1
    for i in range (0,longitud):
        result_temp = inputArray[i] * inputArray[i+1]
        if result < result_temp:
            result = result_temp
            
    return result